README.md

```diff
@@ -42,16 +42,19 @@ test/
 ## GETTING STARTED
 
 ### Requirements
+
 * `conda` >= 3.8.3 - The conda tool can either be installed through [Anaconda](https://www.anaconda.com/products/individual#Downloads) or [Miniconda](https://docs.conda.io/en/latest/miniconda.html).
 * `conda-build` >= 3.20 - Once `conda` is installed, `conda-build` can be installed with the command: `conda install conda-build`
 
 ### CUDA Requirements
+
 If building packages that use CUDA, a tar package of TensorRT 7.0 for CUDA 10.2 will need to be [downloaded](https://developer.nvidia.com/nvidia-tensorrt-7x-download) ahead of time. The downloaded file should be placed in a new local directory called `local_files`.
 
 Currently CUDA 10.2 is supported by the recipes in Open-CE. The `cudatoolkit` and `cudatoolkit-dev` packages can be sourced from [IBM WML CE](https://public.dhe.ibm.com/ibmdl/export/pub/software/server/ibm-ai/conda/#/).
 
 ### Building a Collection of Packages
-The `build_env.py` script can be used to build a collection of Open-CE packages. An environment file needs to be passed in as input. A selection of environment files are provided within the `envs` directory for different frameworks such as TensorFlow and PyTorch. The output from running `build_env.py` will be a local conda channel (by default called `condabuild`). For more details on `build_env.py`, please see `doc/README.build_env.md`.
+
+The `build_env.py` script can be used to build a collection of Open-CE packages. An environment file needs to be passed in as input. A selection of environment files are provided within the `envs` directory for different frameworks such as TensorFlow and PyTorch. The output from running `build_env.py` will be a local conda channel (by default called `condabuild`). For more details on `build_env.py`, please see [`doc/README.build_env.md`](https://github.com/open-ce/open-ce/blob/master/doc/README.build_env.md).
```

Comment:
Is it OK to add links like this? If we are in a different branch, won't it jump to the master branch?

Reply:
That's true, it will jump to the master branch.
My thought was helping people who landed on the readme for the first time.
Let me see if there's a better way to do it.

Reply:
I think you want to use relative links: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-readmes#relative-links-and-image-paths-in-readme-files

Reply:
That seems to have worked.